lib/v2/screens/transfer/send_enter_data/send_enter_data_screen.dart

```diff
@@ -123,6 +125,8 @@ class SendEnterDataScreen extends StatelessWidget {
                               enteringCurrencyName: currencySeedsCode,
                               autoFocus: state.pageState == PageState.initial,
                             ),
+                            const SizedBox(height: 24),
+                            AlertInputValue('The value exceeds your balance'.i18n, isVisible: state.showAlert),
```

Comment:
we have a translation for this string?

Reply:
I'll do in a next PR this message is something common in theses screens, also I check the figma and Json has a diferent text I thing I should change it to this **Not enough balance**